LOTUS_RELEASE_FLOW.md

```diff
@@ -153,6 +154,9 @@ That said, one can still programmatically get the latest release based on the [B
 * Lotus Node: `git tag -l 'v*' | grep -v "-" | sort -V -r | head -n 1` 
 * Lotus Miner: `git tag -l 'miner/v*' | grep -v "-" | sort -V -r | head -n 1` 
 
+### Why does Lotus still use a `master` branch instead of `main`?
```

Comment:
I intend to put most of this content into https://github.com/filecoin-project/lotus/issues/7356 and just update this text to point there.  I have it inlined here so it could be reviewed.